packages/web3/src/utils/bs58.ts

```diff
@@ -36,11 +36,13 @@ export function isBase58(s: string): boolean {
 }
 
 export function base58ToBytes(s: string): Uint8Array {
+  if (s === '') {
```

Comment:
Maybe we we should allow empty string here?

Reply:
ok

Reply:
Do you mean we should allow empty strings here?

Reply:
I have modified the `base58ToBytes` function to return an empty Uint8Array for an empty string input instead of throwing error.